locations/spiders/tjmaxx.py

```diff
@@ -0,0 +1,131 @@
+# -*- coding: utf-8 -*-
+import scrapy
+import re
+from locations.items import GeojsonPointItem
+
+
+class TjmaxxSpider(scrapy.Spider):
+    name = "tjmaxx"
+    allowed_domains = ["tjx.com", "tjmaxx.com"]
+    start_urls = (
+        'https://tjmaxx.tjx.com/store/stores/allStores.jsp',
+    )
+
+    def parse(self, response):
+        links = response.xpath('//li[@class="storelist-item vcard address"]/a[@href]/@href')
+        for link in links:
+            yield scrapy.Request(
+                response.urljoin(link.extract()),
+                callback=self.parse_links
+            )
+
+
+    def parse_links(self, response):
+        store_details = response.xpath('//form[@id="directions-form"]')
+        for store in store_details:
```

Comment:
You're iterating over this `store_details` list and only keeping the last one. If that's what you meant to do, just operate on the first element of the `store_details` array instead of iterating over it.